src/components/rectificationForm/RectificationForm.tsx

```diff
@@ -66,9 +66,10 @@ const RectificationForm = () => {
     }
   };
 
-  const relations = movedCarFormSelected
-    ? ['driver', 'owner', 'poa-holder']
-    : ['driver', 'owner', 'holder'];
+  const relations =
+    selectedForm === 'parking-fine'
+      ? ['driver', 'owner', 'poa-holder']
+      : ['owner', 'poa-holder'];
```

Comment:
Removing the `driver` from here causes the default value to disappear on moved car form. That should be fixed by for example setting the default value of `vehicleRelation` to be `owner` when on moved car form.